animation keyframes/animation.css

```diff
@@ -0,0 +1,127 @@
+div {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+  }
+  .block {
+    width: 100px;
+    height: 30px;
+    margin-right: 10px;
+    background-color: deepskyblue;
+margin-top: 20px;}
+.redcircle {
+    width: 50px;
+    height: 50px;
+    border-radius: 50px;
+    background-color: crimson;
+    animation-name: Jump;
+    animation-iteration-count: infinite;
+    animation-duration: 2s;
+    animation-direction:alternate;
+    position:absolute;
+}
+.block-down {
+    width: 100px;
+    height: 30px;
+    background-color: deepskyblue;
+}
+@keyframes Jump{
+    from{ margin-top: 0px;}
+    to{ margin-top: 50px;}
+
+}
+.block-down {
+    width: 100px;
+    height: 30px;
+    margin-top: 100px;
+    background-color: deepskyblue;
+}
+.right-block{
+    margin-left: 50px;
+    margin-top: 20px;
+    display: inline-block;
+    width: 30px;
+    height: 100px;
+    background-color: deepskyblue;
+}
+.yellowcircle{ display: inline-block;
+    width: 50px;
+    height: 50px;
+    margin-left: 0px;
+    border-radius: 50px;
+    background-color: yellow;
+    animation-name: Slide;
+    animation-iteration-count: infinite;
+    animation-duration: 1s;
+    animation-direction:alternate-reverse;
+    position:relative;
+}
+@keyframes  Slide{
+    from {left:30px;}
+    to{left: 53px;}
+
+}
+.left-block{
+    margin-top: 20px;
+    display: inline-block;
+     width: 30px;
+    height: 100px;
+    background-color: deepskyblue;
+    position: absolute;
+}
+
+.loading{
+    position:absolute;
+    display:block;
+    margin: 30px;
+    border: 10px dotted #27eb3b;
+    border-radius: 30px;
+    padding:10px;
+    width: 10px;
+    height: 10px;
+    animation: spin 2s linear infinite;
+}
+@keyframes spin {
+    0% { transform: rotate(0deg); }
+    100% { transform: rotate(360deg); }
+}
+.circle {
+    display: inline-block;
+    margin:200px 600px;
+    width: 100px;
+    height: 100px;
+    margin-left: auto);
+    background: radial-gradient(rgb(192, 62, 240), rgb(156, 237, 233));
+    border-radius: 100px;
+    animation-name: myRotate;
+    animation-iteration-count: infinite;
+    animation-duration: 2s;
+    animation-direction:normal;
+    position: absolute;
```

Comment:
Please don't use `absolute` here, just put it close to other components. Feel free to use margins to separate them visually.